mender-convert-extract

```diff
@@ -93,9 +93,14 @@ done
 # but we do require it be present and hence we might need to generate
 # a filesystem image here.
 if [ ${nr_of_parts} -eq 1 ]; then
-  log_info "Generating boot partition (required, does not exist in original image)"
-  run_and_log_cmd "dd if=/dev/zero of=work/boot-generated.vfat count=${MENDER_BOOT_PART_SIZE_MB} bs=1M status=none"
-  run_and_log_cmd "${MKFS_VFAT} work/boot-generated.vfat"
```

Comment:
The `boot-generated.vfat` name is used `modules/disk.sh`, so please don't change it unless you also change the logic there.

Reply:
True. The logic change here basically should make the special cases around `boot-generated.vfat` obsolete. Should I just remove the `disk_boot_part` and `disk_root_part` functions altogether ?

Reply:
True. The logic change here basically should make the special cases around boot-generated.vfat obsolete. Should I just remove the disk_boot_part and disk_root_part functions altogether ?

Reply:
I *think* that should be ok. I'm not an expert on this code, so it would be nice if @mirzak could confirm this.

Reply:
I am refreshing my memory on this , hold on a sec